### PR TITLE
Rough out event card

### PR DESCRIPTION
**Pull request by abegong**

### CODE REVIEW

**Comment by ledwards** [comment]
Have some suggestions. Did you want to give me commit access, or should I fork and PR?

**Comment by abegong** [comment]
Just added you to the repo.

Abe Gong, PhD
CEO and Cofounder, superconductive.com ( http://superconductive.com )
Core contributor, greatexpectations.io ( http://greatexpectations.io )

Sent via Superhuman ( ***@***.*** )

File: events.html [hunk 1]
```diff
@@ -43,8 +43,42 @@
...
                         <div class="card-body">                
-                            June 2021
-                            <h3>TBD</h3>
+                            Th, Jun 24 : 6:00 - 9:00, some location
+                            <h3>How technically-savvy VCs think about startups</h3>
+                            <hr/>
+                            <p>
+                                Our panelists are all venture capitalists who invest in technical startups. We will
+                                discuss the industry as they see it, the technologies they're excited about, advice
+                                for foundners and prospective founders in technical industries, and field your questions.
```
Comment (abegong) [inline events.html]:
```suggestion
                                for founders and prospective founders in technical industries, and field your questions.
```

File: events.html [hunk 2]
```diff
@@ -43,8 +43,42 @@
...
+                                <ul>
+                                    <li><strong>6:00</strong> Doors open, socializing, mingling, and snacking</li>
+                                    <li><strong>7:00</strong> Panel: How technically-savvy VCs think about startups</li>
+                                    <li><strong>7:30</strong> Panel Q&A</li>
+                                    <li><strong>7:50</strong> Mingle, socialize</li>
+                                    <li><strong>9:00</strong> Doors close</li>
+                                </ul>
+                            </p>
+                            <p>
+                                Admission is one PR accepted into <a href="wisdom.html">/wisdom</a>.<br/>
```
Comment (abegong) [inline events.html]:
```suggestion
```

On reflection, I don't think we want to gatekeep that strongly. We can encourage contributions at the meetup itself.

File: events.html [hunk 3]
```diff
@@ -43,8 +43,42 @@
...
+                            <p>
+                                Our panelists are all venture capitalists who invest in technical startups. We will
+                                discuss the industry as they see it, the technologies they're excited about, advice
+                                for foundners and prospective founders in technical industries, and field your questions.
+                            </p>
+                            <p>
+                                Panelists:
+                            </p>
+                            <ul>
+                                <li>Sarah Catanzaro, Amplify Partners</li>
```
Comment (abegong) [inline events.html]:
```suggestion
                                <li>TBD</li>
```

File: events.html [hunk 4]
```diff
@@ -43,8 +43,42 @@
...
+                                <li>Naomi Ionita, Menlo Ventures</li>
```
Comment (abegong) [inline events.html]:
```suggestion
                                <li>TBD</li>
```

File: events.html [hunk 5]
```diff
@@ -43,8 +43,42 @@
                 <div class="col-8">
                     <div class="card mb-2">
                         <div class="card-body">                
-                            June 2021
-                            <h3>TBD</h3>
+                            Th, Jun 24 : 6:00 - 9:00, some location
```
Comment (abegong) [inline events.html]:
```suggestion
                            Th, Jun 24 : 6:00 - 9:00, location TBD
```

File: events.html [hunk 6]
```diff
@@ -43,8 +43,41 @@
...
+                            </p>
+                            <ul>
+                                <li>TBD</li>
+                                <li>TBD</li>
+                                <li>Randall Lloyd, Grix</li>
+                                <li>Bryan Offut, Index Ventures</li>
+                                <li>Lee Edwards, Root Ventures (Moderator)</li>
+                            </ul>
+                            <p>
+                                Food will be served
```
Comment (abegong) [inline events.html]:
```suggestion
                                Food will be served. Masks and distancing will follow state and city guidelines.
```

File: events.html [hunk 7]
```diff
@@ -43,8 +43,41 @@
...
+                            </p>
+                            <p>
```
Comment (abegong) [inline events.html:67]:
```suggestion
```

File: events.html [hunk 8]
```diff
@@ -43,8 +43,41 @@
...
+                                Venue is outdoors. Masks and distancing will follow state guidelines.<br/>
```
Comment (abegong) [inline events.html]:
```suggestion
```

File: events.html [hunk 9]
```diff
@@ -43,8 +43,39 @@
...
+                                <li>TBD</li>
+                                <li>TBD</li>
+                                <li>Randall Lloyd, Grix</li>
+                                <li>Bryan Offut, Index Ventures</li>
+                                <li>Lee Edwards, Root Ventures (Moderator)</li>
+                            </ul>
+                            <p>
+                                Food will be served. Masks and distancing will follow state and city guidelines.
+                            </p>
+                                <ul>
```
Comment (abegong) [inline events.html:68]:
```suggestion
                            <p>
                                <ul>
```

**Pull request merged**